### PR TITLE
create community tab + pages

### PR DESCRIPTION
**Pull request by isabelizimm**
Create community tab, bring in community metrics and code of conduct to be easily found on the site.

### CODE REVIEW

**Review by oindrillac — APPROVED** [review]
/lgtm 👍 
/approve

**Review by tumido — APPROVED** [review]
/lgtm
/approve

File: content/community/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### **Who are we?**
+
+We are data scientists, software engineers and DevOps professionals working within the Operate First framework on open source software with open infrastructure, focused on solving large scale operational issues around developing intelligent applications and managing hybrid cloud systems through the application of advanced automation and machine learning.
+
+### **Join us!**
+
+Get in touch with us via [Slack](https://join.slack.com/t/operatefirst/shared_invite/zt-o2gn4wn8-O39g7sthTAuPCvaCNRnLww), follow our [Twitter](https://twitter.com/operatefirst), or view community project demos and updates on our [YouTube](https://www.youtube.com/channel/UCe87bwqlGoBQs2RvMQZ5_sg)
+
+If you're interested in data science, check out our [data science projects](https://www.operate-first.cloud/data-science/projectsoverview.md). If you are more interested in site reliabilty engineering, we have [SRE resources](https://www.operate-first.cloud/operations/sre/README.md) to help you get started!
```
Comment (durandom) [inline content/community/index.md]: let's make the links relative, ie. remove the `https://www.operate-first.cloud/` portion

**Review by durandom — CHANGES_REQUESTED** [review]

**Comment by isabelizimm** [comment]
@tumido @oindrillac bumping this :)

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm
/approve

**Comment by tumido** [comment]
Prow not merging this repo, so I'm gonna merge it manually. :slightly_smiling_face:

**Pull request merged**